.circleci/config.yml

```diff
@@ -94,9 +94,23 @@ jobs:
           name: "Check that there are no generated changes"
           command: make check-no-changes
 
+  deploy_test:
+    executor: ubuntu_vm
+    environment:
+      NUODB_CP_URL_BASE: "https://dev.us-east-1.dbaas.nuodb.io/api"
+    steps:
+     - checkout
+     - run:
+        command: |
```

Comment:
Can you give this a name so that it shows that instead of the command in the CircleCI UI?

Reply:
Sure.